packages/core/src/core/index.ts

```diff
@@ -26,7 +27,7 @@ export interface Core {
   },*/
 
   Repository: {
-    list: (token: string) => Promise<Communication.RepoData>;
```

Comment:
Is the `Communication.RepoData` variable is  still used ? If no you can remove it :)

Reply:
same here, validateSession is still used so RepoData as well